client/reader/notifications/controller.jsx

```diff
@@ -11,7 +11,7 @@ export function notifications( context, next ) {
 	const basePath = sectionify( context.path );
 	const mcKey = 'notifications';
 	const state = context.store.getState();
-	const shouldShowGlobalSidebar = getShouldShowGlobalSidebar( state, null, 'reader' );
+	const shouldShowGlobalSidebar = getShouldShowGlobalSidebar( state, null, 'reader', null );
```

Comment:
Any reason for the `null` here? Would the name be 'notifications' at this point or something? Also unsure why we are hardcoding values in this file as opposed to using currentSection?.group/name as we are in the others?